matrix-sdk/src/main/java/org/matrix/androidsdk/rest/model/message/Message.java

```diff
@@ -26,6 +26,10 @@
     public static final String MSGTYPE_FILE = "m.file";
     public static final String FORMAT_MATRIX_HTML = "org.matrix.custom.html";
 
+    // Add, in local, a fake message type in order to StickerMessage can inherit Message class
+    // Because sticker isn't a new message type but a new event type without msgtype field
```

Comment:
Can we remove "new" in the comment?
Stickers will be no more new next time we play(code) with them.